tester_app/client/app/bundles/HelloWorld/components/HelloWorldWidget.jsx

```diff
@@ -0,0 +1,32 @@
+import React, { PropTypes } from 'react';
+import Immutable from 'immutable';
+
+// Super simple example of the simplest possible React component
+export default class HelloWorldWidget extends React.Component {
+  static propTypes = {
+    actions: PropTypes.object.isRequired,
+    $$helloWorldStore: PropTypes.instanceOf(Immutable.Map).isRequired,
+  }
+
+  _handleChange() {
+    const name = this.refs.name.value;
+    this.props.actions.updateName(name);
+  }
+
+  render() {
+    const $$helloWorldStore = this.props.$$helloWorldStore;
+    const _handleChange = ::this._handleChange;
+    const name = $$helloWorldStore.get('name');
```

Comment:
@alexfedoseev is it worth commenting on why we name is not "immutable" because it's a plain JS String? Maybe not.